provisioning/image_customization_test.go

```diff
@@ -116,33 +122,65 @@ func TestNewImageCustomizationContainer(t *testing.T) {
 		"IRONIC_RAMDISK_SSH_KEY": "sshkey",
 	}
 
+	container4 := corev1.Container{
+		Name: "image-customization-controller",
+		Env: []corev1.EnvVar{
+			{Name: "DEPLOY_ISO", Value: "/shared/html/images/ironic-python-agent.iso"},
+			{Name: "DEPLOY_INITRD", Value: "/shared/html/images/ironic-python-agent.initramfs"},
+			{Name: "IRONIC_BASE_URL", Value: "https://192.168.0.2:6385"},
+			{Name: "IRONIC_AGENT_IMAGE", Value: "registry.ci.openshift.org/openshift:ironic-agent"},
+			{Name: "REGISTRIES_CONF_PATH", Value: "/etc/containers/registries.conf"},
+			{Name: "IP_OPTIONS", Value: "ip=dhcp"},
+			{Name: "ADDITIONAL_NTP_SERVERS", Value: "192.168.1.252,192.168.1.253"},
+			{Name: "IRONIC_RAMDISK_SSH_KEY", Value: "sshkey"},
+		},
+		VolumeMounts: expectedVolumeMounts,
+	}
+	secret4 := map[string]string{
+		"IRONIC_BASE_URL":        "https://192.168.0.2:6385",
+		"IRONIC_AGENT_IMAGE":     "registry.ci.openshift.org/openshift:ironic-agent",
+		"IRONIC_RAMDISK_SSH_KEY": "sshkey",
+	}
+
 	tCases := []struct {
 		name              string
 		ironicIPs         []string
 		proxy             *v1.Proxy
 		expectedContainer corev1.Container
 		expectedSecret    map[string]string
+		ntpServers        []string
 	}{
 		{
 			name:              "image customization container with proxy",
 			ironicIPs:         []string{ironicIP},
 			proxy:             testProxy,
 			expectedContainer: container1,
 			expectedSecret:    secret1,
+			ntpServers:        []string{},
```

Comment:
nit: you don't need to provide this value: Go uses the empty value by default.

Reply:
technically it uses `nil` by default for a slice